modules/mobile/app/controllers/mobile/v0/appointments_controller.rb

```diff
@@ -94,12 +95,15 @@ def fetch_appointments(validated_params)
         appointments
       end
 
-      def paginate(appointments, validated_params)
-        appointments = appointments.filter do |appointment|
+      def filter_by_date_range(appointments, validated_params)
```

Comment:
This should never have been in the paginate method. I probably just failed to move it out when I did a refactor months ago.